README.md

```diff
@@ -11,3 +11,41 @@ Health Check
 ---
 
 To see your applications health enter url `http://localhost:8081/healthcheck`
+
+
+ENV Variables
+--- 
+
+To use DropwizardTheITCrowd API you will need to have the following environment variables
+
+- DB_USERNAME &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;# Database Account Username
```

Comment:
This might be better laid out as a table

Reply:
Fixed